relay-server/src/metrics_extraction/transactions/mod.rs

```diff
@@ -833,6 +833,19 @@ mod tests {
                         "db.system": "MyDatabase",
                         "db.operation": "SELECT"
                     }
+                },
+                {
+                    "description": "GET cache:user:{123}",
```

Comment:
Where do the curly braces come from? Do SDKs send these?

Reply:
That's the key for the cache, not dependent on SDKs and dependent on users. I believe curly braces come from users potentially using them from variables, I'm not aware of any other reason.